lib/util/validate.js

```diff
@@ -63,5 +64,21 @@ module.exports = {
   },
   chapter: function (path, data) {
     return this.validate(require('./schema/chapter'), path, data)
+    .then(function (chapter) {
+      if (!countries) {
+        countries = require('../../country_designations')
+      }
+      var cntrs = countries.byGroup
```

Comment:
Did you see the `byCountry` method at the end of `country_designations`?
https://github.com/nodeschool/admin/blob/master/country_designations.js#L271

It seems you have double-implemented it?!